data/management/commands/import_contributors_data.py

```diff
@@ -10,5 +10,6 @@ class Command(BaseCommand):
     IMPORT_DATA = staticmethod(import_data)
 
     def handle(self, *args, **options):
-        for contributor in self.CONTRIBUTORS():
-            self.IMPORT_DATA(contributor)
+        if get_contrib_data():
```

Comment:
`get_contrib_data()` is called and does some work ... then you delete it by not storing it in a variable!!